README.md

```diff
@@ -190,7 +190,7 @@ to be installed and built alongside Elixir:
 ```sh
 # After cloning and compiling Elixir, in its parent directory:
 git clone https://github.com/elixir-lang/ex_doc.git
-cd ex_doc && ../elixir/bin/mix do deps.get + compile
+cd ex_doc && ../elixir/bin/mix do deps.get, compile
```

Comment:
You can do this in the most recent Elixir versions: https://github.com/elixir-lang/elixir/pull/11696

So let's revert this:

```suggestion
cd ex_doc && ../elixir/bin/mix do deps.get + compile
```